gateway_code/nodes.py

```diff
@@ -108,6 +108,23 @@ def status(self):
         """ Status of the node """
         pass  # pragma: no cover
 
+    def verify(self):
+        """ Verifiy the open node """
+        ret_val = 0
+        # Tuple with (class, machine) run 'elftarget.py' on a node firmware
+        if len(self.ELF_TARGET) != 2:
+            ret_val += 1
```

Comment:
This line is not covered by tests. It is possible to add a custom node class in open_nodes_test module to trigger and verify the correct behaviour of this method.

This is the same for line 121 and 125.